std/string.d

```diff
@@ -3144,6 +3252,89 @@ if (isConvertibleToString!Range)
     });
 }
 
+/// Ditto
+auto strip(Range)(Range str, string chars)
+if (isBidirectionalRange!Range && isSomeChar!(ElementEncodingType!Range) &&
+    !isInfinite!Range && !isConvertibleToString!Range)
```

Comment:
drop isInfinite

Reply:
Again, how do you strip characters from the right of an infinite range?

Reply:
With the code as is :)

Reply:
Gotcha.